.github/workflows/integration_tests.yml

```diff
@@ -218,7 +218,7 @@ jobs:
           fi
           python scripts/gha/build_testapps.py \
             --t ${{ needs.check_and_prepare.outputs.apis }} \
-            --u "${{ steps.unity_setup.outputs.unity_version }}" \
+            --u ${{ env.UNITY_VERSION }} \
```

Comment:
Where is this getting set?

Reply:
https://github.com/firebase/firebase-unity-sdk/blob/88d88cba279144e214ea8a4d894250d8d8c52306/gha/unity/action.yml#L50